Vizzuality/landgriffon
# [LANDGRIF-1249]: Wrong behavior in year selector

### General description

This PR fixes the behaviour of the year selector filter on analysis/table

### Testing instructions

- The default last year selected should be the last year with data + 5 years;
- The selectors should allow search values;
- The 'end' year should accept any valid year value bigger then the 'start' year;

## Checklist before merging
